src/redux/configureStore.js

```diff
@@ -10,6 +11,7 @@ const rootReducer = combineReducers({
 
 const store = configureStore({
   reducer: rootReducer,
+  middleware: [thunk],
```

Comment:
- [ ] [optional] It would be nice adding logger middleware to your store, it assists in tracking dataflow and you can easily debug your app using it.